include/deal.II/sundials/kinsol.h

```diff
@@ -214,6 +214,39 @@ namespace SUNDIALS
         picard = KIN_PICARD,
       };
 
+
+      /**
+       * Orthogonalization strategy used for fixed-point iteration with
+       * Anderson acceleration. While `modified_gram_schmidt` is a good default
+       * choice, it suffers from excessive communication for a growing number of
+       * accleration vectors. Instead, the user may use one of the other
+       * strategies. For a detailed documentation consult the SUNDIALS manual.
+       */
+      enum Orthogonalization
+      {
+        /**
+         * The default, stable strategy. Requires an increasing number of
+         * communication steps with increasing Anderson subspace size.
+         * */
+        modified_gram_schmidt = 0,
+
+        /**
+         * Requires two communication steps and a very small linear system
+         * solve.
```

Comment:
I am not sure if the number of communication steps is the most important for the user. A performance-oriented user might find this interesting, but a mathematically interested user might prefer to learn about what the underlying algorithm is doing. For this one, I found
```
Inverse Compact WY Modified Gram Schmidt
```
at the SUNDIALS page, which after some google search explains a bit more
```
Use a compact WY representation in the orthogonalization process of the inverse
iteration with the Householder transformation.
```